services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -180,7 +184,7 @@ def expected_dynamic_sidecar_spec(
                     "paths_mapping": {
                         "inputs_path": "/tmp/inputs",  # noqa: S108
                         "outputs_path": "/tmp/outputs",  # noqa: S108
-                        "state_exclude": ["/tmp/strip_me/*", "*.py"],  # noqa: S108
```

Comment:
Q: any reason to remove "*.py"?

Reply:
Replied [here](https://github.com/ITISFoundation/osparc-simcore/pull/6819/files#r1856505495)